tokio-fs/src/file/clone.rs

```diff
@@ -0,0 +1,37 @@
+use super::File;
+
+use futures::{Future, Poll};
+
+use std::io;
+
+const REPOLL_ERR: &str = "Cannot poll `FileCloneFuture` after it resolves";
+
+/// Future returned by `File::try_clone`.
+/// Clones a file handle into two file handles.
+/// Will panic if polled after returning an item or error.
+#[derive(Debug)]
+pub struct FileCloneFuture {
```

Comment:
I think this could just be named `CloneFuture` no need for file since it sits under the file module.